package.json

```diff
@@ -52,12 +52,19 @@
     "typecheck": "tsc --noemit"
   },
   "dependencies": {
+    "@storybook/components": "^7.2.0 || ^8.0.0",
+    "@storybook/core-events": "^7.2.0 || ^8.0.0",
     "@storybook/design-system": "^7.15.15",
+    "@storybook/manager-api": "^7.2.0 || ^8.0.0",
+    "@storybook/theming": "^7.2.0 || ^8.0.0",
+    "@storybook/types": "^7.2.0 || ^8.0.0",
```

Comment:
Considering you're using the new trick of:
```
import { globalPackages as globalManagerPackages } from "@storybook/manager/globals";
import { globalPackages as globalPreviewPackages } from "@storybook/preview/globals";
```

These storybook dependencies, should be devDependencies.